ts/src/trycp/util.ts

```diff
@@ -23,15 +24,18 @@ export const deserializeTryCpResponse = (response: Uint8Array) => {
  * @param signal - The signal to deserialize.
  * @returns The deserialized signal.
  */
-export const deserializeTryCpSignal = <T>(signal: Uint8Array) => {
-  const deserializedSignal = msgpack.decode(signal);
+export const deserializeTryCpSignal = (signal: Uint8Array) => {
+  const deserializedSignal = msgpack.decode(signal) as RawSignal;
   assertIsSignal(deserializedSignal);
   if (SignalType.App in deserializedSignal) {
-    const {
-      [SignalType.App]: { cell_id, signal: payload, zome_name },
-    } = deserializedSignal;
-    const decodedPayload = msgpack.decode(payload) as T;
-    return { cell_id, payload: decodedPayload, zome_name };
+    const { cell_id, signal, zome_name } = deserializedSignal[SignalType.App] as EncodedAppSignal;
+    const decodedPayload = msgpack.decode(signal);
+    const app_signal: AppSignal = {
+      cell_id,
+      zome_name,
+      payload: decodedPayload,
+    };
+    return { App: app_signal } as Signal;
```

Comment:
Please replace the type assertions by simply updating `assertIsSignal` to `assertIsRawSignal`, which should `asserts signal is RawSignal`.